pysisu/pysisu.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2022 Sisu Data, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from io import UnsupportedOperation
+from typing import Union
+from urllib import request
+from pysisu.formats import LatestAnalysisFormats
+from pysisu.query_helpers import build_url, pathjoin
+from pysisu.sisu.v1.api import LatestAnalysisResultResponse
+
+
+class Table:
+    # TODO move over from query_wrapper
+    pass
+
+
+class PySisu:
+    '''
```

Comment:
i think we use the triple quotes in python code in the rest of the code base. are you using any formatter like black for consistency?